action.yaml

```diff
@@ -94,6 +87,12 @@ runs:
       shell: bash
       run: pip3 install -r ./packj/requirements.txt
 
+    - name: Move user-defined .packj.yaml to the cloned `packj` folder
+      if: env.CHECK_PASSED == 'true'
+      shell: bash
+      run: |
+        rm ./packj/.packj.yaml
+
```

Comment:
I think you were testing something with this code. Let's fix this. Rest looks good.